Airship/AirshipCore/Source/DefaultLogHandler.swift

```diff
@@ -0,0 +1,44 @@
+/* Copyright Airship and Contributors */
+
+import Foundation
+import os
+
+/**
+ * Default log handler. Logs to either os.Logger or just prints depending on OS version.
+ */
+class DefaultLogHandler: AirshipLogHandler {
+    @available(macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0, *)
+    static var logger: Logger = Logger(subsystem: Bundle.main.bundleIdentifier ?? "", category: "Airship")
```

Comment:
Static here. I tried using a lazy computed property but it still caused an issue on iOS 13 and older. Static properties seem fine (they are also lazy).

Reply:
@rlepinski 

Another trick from my "log wrapper" type. 

```

    private var _logger: Any?
    
    @available(iOS 14.0, *)
    private var logger: Logger {
        // swiftlint:disable:next force_cast
        return _logger as! Logger
    }

    private init(subsystem: String, category: String) {
        if #available(iOS 14.0, *) {
            _logger = Logger(subsystem: subsystem, category: category)
        } else {
            // This implementation will do nothing for iOS < 14
            // Implementation of log funcs might make use of Legacy OSLog API but then that presents the 
            // StaticString issue in the API call site
        }
    }
```

I don't like it and I will remove from my code the moment I can drop iOS 13 support.... but it gets the job done. If you plan to keep it as static var, I'd at least make it private or private(set). It doesn't fix the immutability problem, but it prevents internal classes from trying to change it (even though they wouldn't). I just like protecting code that way.

Reply:
Yeah it should be private let. Not sure which one i like better. You can at least define the subsystem and category on the fly, not that we will. I'll probably just keep it static for now until I have the need for that. Thanks!